services/slam/server.go

```diff
@@ -157,7 +158,35 @@ func (server *subtypeServer) GetPointCloudMapStream(req *pb.GetPointCloudMapStre
 func (server *subtypeServer) GetInternalStateStream(req *pb.GetInternalStateStreamRequest,
 	stream pb.SLAMService_GetInternalStateStreamServer,
 ) error {
-	return errors.New("unimplemented stub")
+	ctx := context.Background()
+	ctx, span := trace.StartSpan(ctx, "slam::server::GetInternalStateStream")
+	defer span.End()
+
+	svc, err := server.service(req.Name)
+	if err != nil {
+		return err
+	}
+
+	f, err := svc.GetInternalStateStream(ctx, req.Name)
+	if err != nil {
+		return err
+	}
+	for {
+		rawChunk, err := f()
+
+		if errors.Is(err, io.EOF) {
+			return nil
```

Comment:
We need to confirm whether or not in this condition rawChunk can contain content or not.

Reply:
discussed in above comment